### PR TITLE
Use 'verbose' listr renderer in CI

### LINKED ISSUE #465: Integration test need refactor
never able to scroll to the end on CI

https://travis-ci.org/sindresorhus/eslint-plugin-unicorn/jobs/619179777?utm_medium=notification&utm_source=github_status

even raw log is colored

https://api.travis-ci.org/v3/job/619179777/log.txt

really hard to debug

### PR DESCRIPTION
**Pull request by futpib**
fixes #465

### CODE REVIEW

**Comment by futpib** [comment]
[Here is the new log](https://travis-ci.org/sindresorhus/eslint-plugin-unicorn/jobs/619266129?utm_medium=notification&utm_source=github_status) ([raw](https://api.travis-ci.org/v3/job/619266129/log.txt)), still has color codes in it, but much better than the [one posted in the issue](https://travis-ci.org/sindresorhus/eslint-plugin-unicorn/jobs/619179777?utm_medium=notification&utm_source=github_status) ([raw](https://api.travis-ci.org/v3/job/619179777/log.txt)).

**Comment by fisker** [comment]
yeah much better , could you remove this line to see a fail log 

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/bb53614324e40dfc58cb332c9f6cbaef0558f593/rules/regex-shorthand.js#L17

**Comment by fisker** [comment]
also, please change this codition to `$SKIP_TEST != true`

 https://github.com/sindresorhus/eslint-plugin-unicorn/blob/bb53614324e40dfc58cb332c9f6cbaef0558f593/.travis.yml#L19

I though integration test generate coverage report, when I change it last time.

**Comment by fisker** [comment]
about color would this help?

https://github.com/chalk/chalk/blob/master/readme.md#chalklevel

if not,  output is good enough, I think

**Comment by fisker** [comment]
I created a failed log on my fork 

https://travis-ci.com/fisker/eslint-plugin-unicorn/jobs/261974089

**Comment by fisker** [comment]
I'm trying to make it print more useful info

**Comment by fisker** [comment]
the new `stderr` can be useful 

```diff
 decode-uri-component (eslint --format json --config /home/travis/build/fisker/eslint-plugin-unicorn/test/integration/eslint-config-unicorn-tester/index.js /tmp/2f42f36a963583537f6e1a5ee262be22 --fix-dry-run)
Command failed with exit code 1: npx eslint --format json --config /home/travis/build/fisker/eslint-plugin-unicorn/test/integration/eslint-config-unicorn-tester/index.js /tmp/2f42f36a963583537f6e1a5ee262be22 --fix-dry-run
+Invalid regular expression: /{{a-bad-patter-to-print-error:::\+}}/u: Lone quantifier brackets
+Occurred while linting /tmp/2f42f36a963583537f6e1a5ee262be22/index.js:85
```

full log: 

https://travis-ci.com/fisker/eslint-plugin-unicorn/jobs/262025663#L458

compare to old one

https://travis-ci.com/fisker/eslint-plugin-unicorn/jobs/261974089#L603

**Comment by fisker** [comment]
//cc @futpib @sindresorhus

File: test/integration/test.js [hunk 1]
```diff
@@ -130,7 +131,10 @@ const execute = name => {
 			title: 'Clean up',
 			task: () => del(destination, {force: true})
 		}
-	], {
+	].map(({title, ...task}) => ({
+		title: [name, title].join(' / '),
+		...task
+	})), {
```
Comment (fisker) [inline test/integration/test.js]:
```suggestion
	].map(({title, task}) => ({
		title: `${name} / ${title}`,
		task
	})), {
```

**Pull request merged**